picli/sast/base.py

```diff
@@ -101,14 +119,21 @@ def zip_files(self, destination):
             f'{destination}/{self.name}.zip', 'w', zipfile.ZIP_DEFLATED
         )
         for file in self.run_config.files:
-            if file['sast'] == f'{self.name}':
-                if self.config.debug:
-                    message = f'Writing file: {file["file"]} to zip archive'
-                    LOG.info(message)
-                zip_file.write(
-                    f"{self.config.base_config.base_dir}/{file['file']}",
-                    file['file']
-                )
+            if self.pipe_config.debug:
+                message = f'Writing {file["file"]} to zip'
+                LOG.info(message)
+            zip_file.write(
+                file['file'],
+                os.path.relpath(file['file'], self.pipe_config.base_config.base_path)

```

Comment:
I think we should have "util.good_path" to handle all pathing the same

Reply:
Agreed, I think I'll save that for a separate PR.